internal/blocktest/test_hash_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 The go-ethereum Authors
```

Comment:
Please omit this file, no need to test this

Reply:
done